tests/test_build_recipe.py

```diff
@@ -426,6 +436,7 @@
                 entrypoint=["/usr/bin/emacs"],
                 cmd=["/usr/bin/gcc"],
                 maintainer="invalid@suse.com",
+                logo_url="https://suse.com/assets/emacs-logo.svg",
```

Comment:
I wonder if we need to move that to a repository e..g under opensource.suse.com/bci/assets/ and maintain it there..

Reply:
Couldn't that make us liable for the content and the licenses? If we just link to it, we should be good. IANAL applies of course

Reply:
most logos are CC, so you only have trademark. and you have that issue in either case.